src/atoms/Button.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { FC, ButtonHTMLAttributes } from "react";
+import styled from "styled-components";
+
+export enum ButtonColors {
+  PRIMARY = "#0D5474",
+  SECONDARY = "#F9B759",
```

Comment:
These colors should be OnlineBlue and OnlineOrange from style

Reply:
Sadly TS enums can't use computed values (variables/functions). So the solution was a bit uglier, but it works.